applications/jupyterhub/deploy/templates/hub/deployment.yaml

```diff
@@ -31,14 +31,13 @@ spec:
         {{- end }}
     spec:
       {{- if .Values.local }}
-      {{ $domain := .Values.domain }}
       hostAliases:
         - ip: {{ .Values.localIp }}
           hostnames:
           {{ printf "- %s" .Values.domain }}
           {{- range $app := .Values.apps }}
           {{- if $app.harness.subdomain }}
-          {{ printf "- %s.%s" $app.harness.subdomain $domain }}
+          {{ printf "- %s.%s" $app.harness.subdomain "mnp.local" }}
```

Comment:
please look at this commit... https://github.com/MetaCell/cloud-harness/commit/fbef8934d212959ef4c49bc332a26d38a3a039d3

mnp.local is wrong :-)